extensions/wikia/UserLogin/js/MarketingOptIn.js

```diff
@@ -12,21 +12,21 @@ define('usersignup.marketingOptIn', ['wikia.geo'], function (geo) {
 		var $optInField = wikiaForm.inputs.wpMarketingOptIn,
 			$optInGroup = wikiaForm.getInputGroup('wpMarketingOptIn'),
 			isEurope = geo.getContinentCode() === 'EU',
-			isCanada = geo.getCountryCode() === 'CA';
+			isCanada = geo.getCountryCode() === 'CA',
+			isJapan = geo.getCountryCode() === 'JP';
 
 		if (!$optInField.length || !$optInGroup.length) {
 			throw 'Wikia Form must contain a field called wpMarketingOptIn';
 		}
 
-		if (!isCanada) {
+		if (!isCanada && !isJapan) {
```

Comment:
@lizlux Sorry, I'm late to the party as this ticket passed me by :)
At this point, it'd be nice to have a separate functions for this and for line 26. For instance, I'd suggest `shouldCheckOptIn()`, and `shouldDisplayOptIn()`, for instance.
Having said that, it can wait for now :)